src/components/deals-product-details/DealsProductStoreDetails.tsx

```diff
@@ -97,32 +97,37 @@ const DealsProductStoreDetails = () => {
                                 }
 
 
-                                <Box className="item-details-website" sx={{ display: "flex", mb: "12px" }}>
-                                    <LanguageOutlinedIcon sx={{
-                                        color: theme.palette.text.primary, mr: "12px",
-                                        fontSize: "22px", lineHeight: "16px", fontWeight: 400
-                                    }} />
-                                    <Link href={storeData?.website} target="_blank" className="phone" sx={{
-                                        cursor: "pointer", textDecoration: "none", color: theme.palette.common.black,
-                                        "&:hover": { color: theme.palette.text.primary, transition: ".5s", },
-                                        // width: "100%", whiteSpace: "nowrap", overflow: "hidden", textOverflow: "ellipsis"
-                                    }}>
-                                        <Typography> {storeData?.website}</Typography>
-                                    </Link>
-                                </Box>
+                                {
+                                    storeData?.website === null ? <></> :
```

Comment:
User can use `null` instead of `<></>`
Or You can do like ` {storeData?.website && ( <Box className="item-details-website" sx={{ display: "flex", mb: "12px" }}>....</Box> )}`